src/amo/components/Addon/index.js

```diff
@@ -136,7 +136,6 @@ export class AddonBase extends React.Component {
           className="Addon-theme-header"
           id="Addon-theme-header"
           data-browsertheme={getBrowserThemeData()}
-          ref={(el) => { this.wrapper = el; }}
```

Comment:
This was unused and caused a non-100% coverage error so I deleted it. I know it's unrelated but figured it was worth it.